gen/dcos-config.yaml

```diff
@@ -698,6 +698,21 @@ package:
           "LocalFiles": [
               {
                   "Location": "/opt/mesosphere/active.buildinfo.full.json"
+              },
+              {
+                  "Location": "/opt/mesosphere/active/dcos-metadata/etc/dcos-version.json"
```

Comment:
I just did not realize this information existed in `/opt/mesosphere/etc/dcos-version.json` as well. I'll update the PR